### ISSUE TITLE
Add links on Tails Desktop to documentation and support

### ISSUE THREAD

**Issue by redshiftzero** [freedomofpress/securedrop#2131]
# Description

We should add links on the Desktop (as we do for the SecureDrop source and journalist interface) in the admin and journalist workstations for the SecureDrop Administrator guide and the SecureDrop Journalist guide respectively. On the Admin workstation we should also add a link to the SecureDrop [support portal](https://support.freedom.press).

# User Story

As a SecureDrop user, I want links to documentation and help for when I get confused or stuck.

**Comment by elimisteve** [comment]
Hey @redshiftzero, I'm at a CWC hackathon and someone is trying to do this issue.

Did the Tails people say it's OK to add a new shortcut to the default Tails desktop, or am I misunderstanding when this link should be created?

**Comment by dachary** [comment]
@elimisteve SecureDrop already adds two shortcuts to the tails desktop: 

https://github.com/freedomofpress/securedrop/blob/develop/install_files/ansible-base/roles/tails-config/tasks/create_desktop_shortcuts.yml

**Comment by elimisteve** [comment]
@dachary So this issue is already finished?

**Comment by dachary** [comment]
@elimsteve no, sorry for the confusion, the issue is not resolved. I just meant to point you to examples you can follow :-) These shortcuts are for the journalist and the source interface.

**Comment by zenmonkeykstop** [comment]
Relates to @nathandyer's proposal in #6531 to move to using Gnome shell extensions - if we stick with icons this will get pretty messy, but the menu option would be nice and clean. Keeping this open for now
